source/ddbus/meson.build

```diff
@@ -0,0 +1,37 @@
+lib_src = [
+    'attributes.d',
+    'bus.d',
+    'c_lib.d',
+    'conv.d',
+    'exception.d',
+    'package.d',
+    'router.d',
+    'simple.d',
+    'thin.d',
+    'util.d',
+]
+
+ddbus_lib = library(
+    'ddbus',
+    [lib_src],
+    install: true,
+    version: meson.project_version(),
+    soversion: project_soversion,
```

Comment:
No need since you already have `version`.

Reply:
Ah right